### PR TITLE
Refactor: Move internal command handling to processing host

### PR DESCRIPTION
**Pull request by kft**
## Description
This PR moves all internal command handling to the processing entrypoint and thus, deprecates the InternalCommandDispatcher entrypoint.

### CODE REVIEW

File: source/Energinet.DataHub.MeteringPoints.EntryPoints.Processing/Functions/ScheduleStatus.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 using System;
 
-namespace Energinet.DataHub.MeteringPoints.EntryPoints.InternalCommandDispatcher.Infrastructure.TimerTriggers
+namespace Energinet.DataHub.MeteringPoints.EntryPoints.Processing.Functions
```
Comment (cmgrd) [inline source/Energinet.DataHub.MeteringPoints.EntryPoints.Processing/Functions/ScheduleStatus.cs:17]: Microsoft.Azure.Functions.Worker.Extensions.Timer 4.1.0 contains ScheduleStatus and TimerInfo classes.

File: source/Energinet.DataHub.MeteringPoints.EntryPoints.Processing/Functions/TimerInfo.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-namespace Energinet.DataHub.MeteringPoints.EntryPoints.InternalCommandDispatcher.Infrastructure.TimerTriggers
+namespace Energinet.DataHub.MeteringPoints.EntryPoints.Processing.Functions
 {
     public class TimerInfo
```
Comment (cmgrd) [inline source/Energinet.DataHub.MeteringPoints.EntryPoints.Processing/Functions/TimerInfo.cs:17]: Microsoft.Azure.Functions.Worker.Extensions.Timer 4.1.0 contains ScheduleStatus and TimerInfo classes.

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/InternalCommands/QueuedInternalCommand.cs [hunk 3]
```diff
@@ -34,7 +34,7 @@ public QueuedInternalCommand(Guid id, string type, byte[] data, Instant creation
         public string Type { get;  } = string.Empty;
 
         #pragma warning disable CA1819 // Properties should not return arrays
```
Comment (cmgrd) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/InternalCommands/QueuedInternalCommand.cs:36]: pragma should not be relevant anymore.

File: source/MeteringPoints.EntryPoints.IntegrationTests/Functions/MeteringPointFunctionAppTests.cs [hunk 4]
```diff
@@ -94,10 +94,7 @@ public async Task Metering_point_flow_should_succeed(string testFileXml)
             // Outbox
             await AssertFunctionExecuted(Fixture.OutboxHostManager, "OutboxWatcher").ConfigureAwait(false);
 
-            // InternalCommands
-            await AssertFunctionExecuted(Fixture.InternalCommandDispatcherHostManager, "Dispatcher").ConfigureAwait(false);
```
Comment (cmgrd) [inline source/MeteringPoints.EntryPoints.IntegrationTests/Functions/MeteringPointFunctionAppTests.cs:98]: I think we should still assert that the internal commands are processed. See other comment about setup.

File: source/MeteringPoints.EntryPoints.IntegrationTests/Fixtures/MeteringPointFunctionAppFixture.cs [hunk 5]
```diff
@@ -112,29 +108,22 @@ public async Task InitializeAsync()
             localMessageHubHostSettings.Functions = "BundleDequeuedQueueSubscriber RequestBundleQueueSubscriber";
             localMessageHubHostSettings.Port = ++port;
 
-            internalCommandDispatcherHostSettings.FunctionApplicationPath = $"..\\..\\..\\..\\Energinet.DataHub.MeteringPoints.EntryPoints.InternalCommandDispatcher\\bin\\{buildConfiguration}\\net5.0";
-            internalCommandDispatcherHostSettings.Functions = "Dispatcher";
```
Comment (cmgrd) [inline source/MeteringPoints.EntryPoints.IntegrationTests/Fixtures/MeteringPointFunctionAppFixture.cs:116]: We remove "Dispatcher" here, but I think we should add that to the processingHostSettings instead. (related to the comment about asserting that it runs)

**Review by CBrohus — APPROVED** [review]

**Pull request merged**